### PR TITLE
Fix dependency bugs

### PR DESCRIPTION
**Pull request by NeonDaniel**
Fix bad references to removed or updated dependencies

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,8 @@ def _get_version():
     author='JarbasAI',
     install_requires=["packaging",
                       "ovos_skill_installer>=0.0.5",
-                      "json_database>=0.5.6",
+                      "json_database~=0.5,>=0.5.6",
```
Comment (JarbasAl) [inline setup.py:22]: 0.6.0 should work, can you test? breaking change was on 0.7.0

File: tests/conftest.py [hunk 2]
```diff
@@ -22,12 +22,12 @@ def osm_test(monkeymodule):
...
-    import xdg
+    # import xdg
     import json_database
 
     from ovos_skills_manager import commands, github, licenses, osm, \
         session, skill_entry, upgrade_osm, utils
     import ovos_skills_manager.versioning.osm_versions as versions
     yield
-    print(xdg.BaseDirectory.xdg_config_home)
+    # print(xdg.BaseDirectory.xdg_config_home)
```
Comment (JarbasAl) [inline tests/conftest.py:32]: can you bump ovos_utils and use `ovos_utils.xdg_utils.xdg_config_home()`?

that would also allow bumping json_database

File: setup.py (hunk 1 above)
Comment (NeonDaniel) [reply setup.py:22]: Is there a reason to pin to 0.6 instead of allowing newer versions?

Comment (JarbasAl) [reply setup.py:22]: 0.7.0 moved to combo_lock package and dropped pyxdg
0.6.0 fixed a typo in the Query methods (unused by us) bellow -> below

maybe ovos_utils depended on something introduced in 0.5.6 but probably will work with much much earlier versions

Comment (NeonDaniel) [reply setup.py]: done.

**Pull request merged**